hypertrace-graphql-gateway-service-metric-utils/src/main/java/org/hypertrace/graphql/utils/metrics/gateway/BaselineConvertedAggregationContainer.java

```diff
@@ -0,0 +1,63 @@
+package org.hypertrace.graphql.utils.metrics.gateway;
+
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import lombok.experimental.Accessors;
+import org.hypertrace.core.graphql.common.schema.time.TimeUnit;
+import org.hypertrace.graphql.metric.schema.BaselinedMetricAggregation;
+import org.hypertrace.graphql.metric.schema.BaselineMetricAggregationContainer;
+
+import java.time.Duration;
+import java.util.Map;
+
+import static org.hypertrace.core.graphql.attributes.AttributeModelMetricAggregationType.*;
+import static org.hypertrace.core.graphql.attributes.AttributeModelMetricAggregationType.DISTINCT_COUNT;
+
+@Accessors(fluent = true)
```

Comment:
What does this generate given that the whole interface appears to be implemented below?